flowmachine/flowmachine/core/server/query_schemas/base_exposed_query.py

```diff
@@ -88,35 +64,3 @@ def query_id(self):
         # by flowmachine.core.Query.md5, and the latter is currently being used by
         # the QueryStateMachine, so we need to use it to check the query state.
         return self._flowmachine_query_obj.md5
-
-    @property
-    def query_params(self):
-        """
-        Return the parameters from which the query is constructed. Note that this
-        includes the the parameters of any subqueries of which it is composed.
-
-        Returns
-        -------
-        dict
-            JSON representation of the query parameters, including those of subqueries.
-        """
-        # We need to dynamically import FlowmachineQuerySchema here in order to avoid
-        # a circular import in this file because currently there are dependencies of
-        # the following form (using the example of DailyLocation):
-        #
-        #     DailyLocationSchema -> DailyLocationExposed -> BaseExposedQuery -> FlowmachineQuerySchema -> DailyLocationSchema
-        #
-        # We break the dependency BaseExposedQuery -> FlowmachineQuerySchema here by using
-        # this dynamic import. There is possibly a better way to do this...
-        from flowmachine.core.server.query_schemas import FlowmachineQuerySchema
-
-        return FlowmachineQuerySchema().dump(self)
-
-    def _create_query_info_lookup(self):
```

Comment:
Hmm, I was certain we needed this (or at least _wanted_ to create the query info lookup here) but you are correct that it doesn't seem to be used. As far as I can tell, the query lookup creation currently happens in the run_query action handler. @greenape Just to confirm you think that's the right place for it to happen (so that it's fine to delete this here)?

Reply:
If we turn out to need it, we can put this back in. I figure right now, more of a liability have a dead code path hanging about.